_posts/2020-07-12-ksf-lead-after-first-weekend.md

```diff
@@ -3,7 +3,7 @@ title: Kenilworth Sixth Form lead after the first weekend of matches
 ---
 
 After the first 24 matches of the SR2020 Virtual Competition, team KSF from
-Kenilworth Sixth Form head up the league leaderboard with 58 points. They're
```

Comment:
Thoughts on hyphenating leader-board?

Reply:
no

Reply:
(I think our choice for "livestream" is a bit more tenuous so leaderboard is good)

Reply:
I think I'm in favour of hyphenating and using "leader-board" rather than "leader board".

I've just checked the AP Style Book and their Golfing editor uses 'leaderboard' though most dictionaries I checked have it as two words. Hyphenation feels like a sensible middle ground to me.

Reply:
It does seem mixed online but I'm still pushing for leaderboard 👀
https://dictionary.cambridge.org/dictionary/english/leaderboard
https://www.collinsdictionary.com/dictionary/english/leaderboard

Reply:
Interestingly, Collins seem to recognise leader board as the US version but I haven't seen much else to suggest that
https://www.collinsdictionary.com/dictionary/english/leader-board

Reply:
Given the precedence of deffering to AP for 'livestream' and those dictionary entries, I'm happy for us to settle on leaderboard.

Reply:
For clarity: I prefer the one-word variant "leaderboard" and was surprised that it wasn't in the dictionary.